### ISSUE TITLE
Allow for merging multiselect options

### ISSUE THREAD

**Issue by SolarDesalination** [seatable/seatable#86]
I have three multiselect options; A, B, and C.

I rename C to "A" in the edit options menu. I choose the same colour. I now have three options, A, C, A. So now I have two options with the same name and same colour.

What should happen is if I rename an option and choose the same colour, it should merge the options together.

**Comment by support51** [comment]
Yes same problem I am facing right now, Is there any solution to merge 2 multiselect question in one with 1 option.

**Comment by freeplant** [comment]
We may provide such a feature in the future.
